maven-extension/src/test/java/io/opentelemetry/maven/handler/MojoGoalExecutionHandlerTest.java

```diff
@@ -43,7 +43,7 @@
  * TODO Find a better solution to instantiate a MavenProject and a MojoExecutionEvent. See
  * https://github.com/takari/takari-lifecycle/blob/master/takari-lifecycle-plugin/src/test/java/io/takari/maven/plugins/plugin/PluginDescriptorMojoTest.java
  */
-@SuppressWarnings("DeduplicateConstants")
+@SuppressWarnings({"DeduplicateConstants", "deprecation"})
```

Comment:
From what I could tell, the deprecated call to `MavenSession` constructor is related to the existing TODO

Reply:
You are right. The [takari-plugin-testing-project](https://github.com/takari/takari-plugin-testing-project/) is just [adding support for JUnit5](https://github.com/takari/takari-plugin-testing-project/pull/38), I should be able very soon to cleanup and greatly improve the tests of the Otel Maven Extension.